roles/ovirt-engine-setup/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  author: Tareq Alayan
```

Comment:
I would add Petr Kubica - original developer and maintainer of these

Reply:
ok